src/pydocstyle/checker.py

```diff
@@ -583,41 +618,78 @@ def _check_section(cls, docstring, definition, context):
         if not is_blank(context.previous_line):
             yield violations.D411(capitalized_section)
 
+    @classmethod
+    def _check_numpy_section(cls, docstring, definition, context):
+        """D4{05,06,10,11,13}, D214: Section name checks.
+
+        Check for valid section names. Checks that:
+            * The section name is properly capitalized (D405).
```

Comment:
Please don't repeat all the description from `_check_common_section` as it's likely to go out of date. You can add to "Also yield all the errors from ..." line below.